Developer-guide/Backport-Guidelines.md

```diff
@@ -1,87 +1,26 @@
-Bugs often get fixed in master before release branches. When a bug is
-fixed in the master branch, it might be desirable or necessary in a
-stable branch. To put the fix in the stable branch we need to backport the
-fix to the stable branch.
+As a policy, any new change, bug fix, etc., are to be fixed in 'master'
```

Comment:
In the GlusterFS project, as a policy, any new change, bug fix etc are to be fixed in 'master'

Reply:
Done